InferenceSystem/deploy-aci.yaml

```diff
@@ -12,6 +12,8 @@ properties:
         secureValue: '<cosmos_primary_key>'
       - name: AZURE_STORAGE_CONNECTION_STRING
         secureValue: '<storage_connection_string>'
+      - name: INFERENCESYSTEM_APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
Please also update InferenceSystem/README.md detailing how to configure (where to find instrumentation key, etc.)